assets/translations/en-us.yaml

```diff
@@ -100,6 +105,16 @@ cluster:
       label: Import an existing cluster
       shortLabel: Imported
 
+promptRemove:
+  attemptingToRemove: You are attemping to remove the {type}
+  andOthers: |
+    {count, plural,
+    =0 {.}
+    =1 {, and one other.}
+    other {, and {count} others.}
+    }
+  protip: 'protip: Hold the Command key while clicking delete to bypass this confirmation'
```

Comment:
Almost there.. The appropriate key is utils/platform.js -> `alternateKey`/`alternateLabel`:  meta (== command) for Mac, ctrl for Lin/Win

Reply:
Done